runtime/include/chpl-gpu.h

```diff
@@ -46,6 +46,33 @@ static inline void CHPL_GPU_DEBUG(const char *str, ...) {
   }
 }
 
+#ifdef CHPL_GPU_ENABLE_PROFILE
```

Comment:
Were you planning on including these timers as part of the PR? You don't mention it in the PR description but having some macros/functions for performance debugging seems generally useful. So that makes me wonder if we don't already have these sort of tools somewhere and if not if we should add them.

Anyway, I'm not opposed to not generalizing this and keeping this included in `chpl-gpu.h` directly but you may want to show this to @jhh67 or someone else invested with our runtime libs to see if they have thoughts on it.

Reply:
Offline, Elliot OK'ed this change.